### PR TITLE
MANTA-4629 add Jenkinsfiles to Manta repositories

### PR DESCRIPTION
**Pull request by cburroughs**
artedi had tests and seemed pretty important, so it seemed worthwhile to try something slightly more complicated.

### CODE REVIEW

**Comment by cburroughs** [comment]
Note that the jobs fails because the tests fail!  Catastrophic success! Opened #20

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    post {
+        always {
+            joyMattermostNotification()
```
Comment (KodyKantor) [inline Jenkinsfile]: Does this correspond to a certain channel in mattermost?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '45'))
+        timestamps()
+    }
+
+    stages {
+        stage('top') {
+            parallel {
+                stage('v0.10.48-zone') {
```
Comment (KodyKantor) [inline Jenkinsfile]: If we drop 0.10.48 support do we need this stage?

Comment (cburroughs) [reply Jenkinsfile]: Nope, I'll update with that dropped.

File: Jenkinsfile (hunk 1 above)
Comment (cburroughs) [reply Jenkinsfile]: It defaults to `jenkins` where we have historically sent a bunch of build related automated messages.  You could do ` joyMattermostNotification(channel:'my_cool_hangout'))

**Pull request merged**